addons/sale/controllers/portal.py

```diff
@@ -159,14 +161,21 @@ def portal_order_page(
             'report_type': 'html',
             'backend_url': backend_url,
             'res_company': order_sudo.company_id,  # Used to display correct company logo
+            'link_amount': link_amount,
+            'installment': installment,
         }
 
         # Payment values
-        if order_sudo._has_to_be_paid():
+        if order_sudo._has_to_be_paid() or link_amount:
```

Comment:
Do we really have to support the case when the order is fully paid but the link_amout is set? For now I can not generate a link if an order is fully paid.

Reply:
Yes, currently, if Quote has Online Payment setting set to false it is still possible to generate payment link to it, after discussion with dala, it is a functionality we would like to keep.